Gemfile

```diff
@@ -8,7 +8,7 @@ gem "winrm-fs"
 gem "activesupport", "~> 4.0"
 
 group :test do
-  gem "rake"
+  gem "rake", "< 12"
```

Comment:
We're moving away from finstyle and I didn't realize this repo was still on it so new things to do